### DISCUSSION TITLE
backdrop-filter: blur disappear during the momentum

### DISCUSSION THREAD

**Discussion by byman64** [nolimits4web/swiper#4318] [Q&A]
I have a trouble with the fantastic Swiper.

I am using transparent horizontal boxes with "blur effect" using the css "backdrop-filter: blur(5px);" applied on ::after of div box.

This effect disappear during the "momentum" using the mouse drag and it will appear again when the "auto scroll"  finish.

If I use mouse wheel scrolling the blur never disappear.

[Live Issue](https://www.quakearea.com/mon/#q=QooL-Q3Msk.txt&s=wp-spotbright&b=horizontal)
[`Video to Show what I mean`](https://youtu.be/Egv7lGXW0pY)
Any idea to fix it?

**Comment by evgenii-code** [comment]
I have the same issue with my project when using css property backdrop-filter: blur(10px) on child element of the slide.
Can't figure out what cause it and how to fix this problem

Update: I think this is some Chrome bug, because in Firefox 86.0.1 backdrop-filter works fine (if you enable this feature in about:config page)
Here is my reproduction https://codepen.io/BoringChip/pen/bGByypP

**Comment by TheManFromEarth1** [comment]
I have raised an issue on Chromium:
https://bugs.chromium.org/p/chromium/issues/detail?id=1194050

As a workaround, you have to set transition-duration to 0ms in Swiper. 
In the example, set speed: to 0.

**Comment by Hikk0o** [comment]
Has anyone fixed this?

**Reply by VIPStephan** [reply]
Not here, anyway, since this appears to be a Chromium issue, not a Swiper issue.

**Reply by Hikk0o** [reply]
@VIPStephan Very bad, I use blur in my Electron app and it is very annoying...
